### DISCUSSION TITLE
Prisma 2 "original" argument must be of type function

### DISCUSSION THREAD

**Discussion by bradmsys** [prisma/prisma#2277] [Q&A] [answered]
Hi All
Prisma 2 is looking exciting but having trouble installing.  I could really use some help down here in Australia if anyone can.  
(I'm a bit of a Prisma for Dummy's case)

npx prisma is failing as follows:

```
PS C:\Develop\Git Projects\test6> npx prisma
The "original" argument must be of type function

PS C:\Develop\Git Projects\test6> npm list
test6@1.0.0 C:\Develop\Git Projects\test6
+-- @prisma/cli@2.0.0-beta.3
+-- @types/node@13.13.2
+-- node@13.13.0
| `-- node-bin-setup@1.0.6
+-- ts-node@8.9.0
| +-- arg@4.1.3
| +-- diff@4.0.2
| +-- make-error@1.3.6
| +-- source-map-support@0.5.18
| | +-- buffer-from@1.1.1
| | `-- source-map@0.6.1
| `-- yn@3.1.1
`-- typescript@3.8.3

**Comment by ryands17** [comment, marked as answer]
Hey @bradmsys :wave: 
Could you try running `npx prisma init` and let me know if that works?

And also I would recommend installing `@prisma/cli` locally as global installation wouldn't be required.

**Comment by asmusj224** [comment]
Check your node version. I was having this issue until I started using 12.4.0

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
